lecture_1/test/PureFunctionsTest.js

```diff
@@ -1,37 +1,38 @@
-import { merge, associate } from "../src/PureFunctions";
+import { merge, associate } from '../src/PureFunctions';
```

Comment:
Why the formatting changes?
Lecture 3 introduces prettier properly - which you probably should consider bringing up to lecture 2/1.
Currently,
```shell
yarn add --dev prettier
cp ../lecture_3/.prettier* .
npx prettier --write .
```
would undo all the single-quote changes.